enterprise/static/enterprise/sass/partials/utilities/_variables.scss

```diff
@@ -1 +1,6 @@
 $blue: #126F9A;
+$margin-xs: 5px;
+$margin-sm: 10px;
+$margin-md: 15px;
+$margin-lg: 20px;
+$margin-xl: 40px;
```

Comment:
Do we need these variables? Bootstrap/Paragon have spacing utilities we probably could be using from the theme itself.

For example, instead of using these variables for `margin-right: 10px`, we might want to use `@extend .mr-1;`.

Reply:
Will address in follow-up ticket.

Reply:
https://openedx.atlassian.net/browse/ENT-3823